scrapyloganalyzer/__init__.py

```diff
@@ -204,4 +204,22 @@ def error_rate(self):
         :returns: an estimated lower bound of the true error rate
         :rtype: float
         """
-        return self.item_counts["FileError"] / (self.item_counts["File"] + self.item_counts["FileError"])
+        error_count = self.item_counts["FileError"]
+        if "invalid_json_count" in self.logparser["crawler_stats"]:
+            error_count += self.logparser["crawler_stats"]["invalid_json_count"]
+        return error_count / (self.item_counts["File"] + error_count)
+
+    @property
+    def drop_rate(self):
```

Comment:
Let's use type annotations instead of docstrings. You can make the same change to other methods.

```suggestion
    def drop_rate(self) -> float:
```